packages/protocol/scripts/bash/check-versions.sh

```diff
@@ -38,5 +38,5 @@ yarn build:sol
 rm -rf $BUILD_DIR_2 && mkdir -p $BUILD_DIR_2
 mv build/contracts $BUILD_DIR_2
 
-CONTRACT_EXCLUSION_REGEX=".*Test.*|I[A-Z].*|ReleaseGold"
+CONTRACT_EXCLUSION_REGEX=".*Test.*|I[A-Z].*|ReleaseGold|.*LinkedList.*"
```

Comment:
Why exclude them from the report list? Is it necessary?

Reply:
This will allow us to upgrade to solidity 0.5.12 so that we can use the `chainId` instruction for the MetaTransactionWallet. Unfortunately, the upgrade to 0.5.12 results in some very minor bytecode changes to these libraries, that we're happy to ignore so long as the source code doesn't change.